charts/ix-chart/2208.0.0/templates/_workload.tpl

```diff
@@ -60,6 +60,8 @@ containers:
   stdin: {{ .Values.stdin }}
   securityContext:
     privileged: {{ .Values.securityContext.privileged }}
+    runAsUser: {{ .Values.securityContext.runAsUser }}
```

Comment:
Let's please make this optional i.e if user has not specified any overrides, we should not specify a default which we are doing right now for root and let the image uid/gid take preference. Thanks!